File: ruby/1 - Iniciante/1015.rb
class Ponto
	attr_reader :x, :y
	def initialize(x, y)
		@x = x.to_f
		@y = y.to_f
	end
end

def distancia(pontoA, pontoB)
	'%.4f' % Math.sqrt((pontoA.x - pontoB.x)**2 + (pontoA.y - pontoB.y)**2)
end

entrada = gets.chomp.split(" ")
pontoA = Ponto.new(entrada[0], entrada[1])
entrada = gets.chomp.split(" ")
pontoB = Ponto.new(entrada[0], entrada[1])
puts distancia(pontoA, pontoB)

File: python3/1015.py
import math

class Ponto:
    def __init__(self, x, y):
        self.x = float(x)
        self.y = float(y)

def dist(a, b):
    return math.sqrt((a.x - b.x)**2 + (a.y - b.y)**2)

ponto1 = Ponto(*input().split())
ponto2 = Ponto(*input().split())
print(format(dist(ponto1, ponto2), '.4f'))
